java/code/src/com/redhat/rhn/manager/contentmgmt/ContentManager.java

```diff
@@ -81,9 +85,23 @@
 public class ContentManager {
 
     private static final String DELIMITER = "-";
+    private static final Logger LOG = Logger.getLogger(ContentManager.class);
+    private ModulemdApi modulemdApi;
 
-    // forbid instantiation
-    private ContentManager() { }
+    /**
+     * Initialize a new instance
+     */
+    public ContentManager() {
+        this(null);
```

Comment:
It just calls the second constructor with a `null` as the `modulemdApi` parameter.
Since the initialization is in the second one, I just chained constructors.

Reply:
ok